foundation-models/openai/src/main/java/com/sap/ai/sdk/foundationmodels/openai/model/OpenAiChatCompletionOutput.java

```diff
@@ -25,4 +26,26 @@ public class OpenAiChatCompletionOutput extends OpenAiCompletionOutput {
   @JsonProperty("system_fingerprint")
   @Getter(onMethod_ = @Nonnull)
   private String systemFingerprint;
+
+  public void addDelta(OpenAiChatCompletionOutput delta) {
```

Comment:
(Major)

What do you need this method for? It is unused or is it not? Certainly missing JavaDoc and there's still a TODO. I assume this is a left-over(?)